server/web/test/index.hbs

```diff
@@ -123,7 +123,7 @@
         {{#compare visible "y" operator="==="}}
         {{!-- TODO: add "or admin session" --}}
           <li {{#compare revision ../revision operator="==="}}class="current"{{/compare}}>
-            <a href="/{{slug}}{{#compare 1 revision}}/{{revision}}{{/compare}}">Revision {{revision}}</a>: published {{formatAuthor author authorURL}}
+            <a href="/{{../../page.slug}}{{#compare 1 revision}}/{{revision}}{{/compare}}">Revision {{revision}}</a>: published {{formatAuthor author authorURL}}
```

Comment:
it'd be great to add a description for the Pull Request or a comment here describing how this fixes the link. Good learning opportunity :)